yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/table-columns-data-types.psql

```diff
@@ -1,14 +1,17 @@
 -- gathering schema_name, table_name, column_name, data_type
 CREATE TEMP TABLE temp_column_info AS
 SELECT
-    table_schema AS schema_name,
-    table_name,
-    column_name,
-    data_type
+    c.table_schema AS schema_name,
+    c.table_name,
+    c.column_name,
+    CASE
+        WHEN c.data_type = 'USER-DEFINED' THEN c.udt_name
```

Comment:
I'm confused about how this fix is relevant here? 
It looks like we are just getting the type name from the SQL statement parse tree, checking if it is part of the list of PostgresUnsupportedDataTypes, that's it. RIght?

Reply:
This is relevant for reporting these postGis unsupported datatypes like `GEOMETRY / GEOGRAPHY` in Assessment report, as these are user-defined type from extension so it was earlier getting stored as `USER-DEFINED` and we were not able to report these there. So I have modified the query to return the exact udt type name in the data.